travis/mypyer

```diff
@@ -0,0 +1,2 @@
+MYPYPATH="smac/configspace/ smac/epm/ smac/facade/ smac/initial_design smac/intensification smac/optimizer smac/runhistory smac/scenario smac/stats smac/tae smac/utils"
+mypy --ignore-missing-imports $MYPYPATH
```

Comment:
We could try to be more strict: `--disallow-any`, `--disallow-untyped-defs`, `--strict`. What do you think?

Reply:
I'm actually not sure if we want this. Additional run info is of type `Any`, right?

Reply:
Done